pkg/machine/wsl/wutil/wutil.go

```diff
@@ -5,14 +5,69 @@ package wutil
 import (
 	"bufio"
 	"io"
+	"os"
 	"os/exec"
+	"path/filepath"
 	"strings"
+	"sync"
 	"syscall"
 
 	"golang.org/x/text/encoding/unicode"
 	"golang.org/x/text/transform"
 )
 
+var (
+	once    sync.Once
+	wslPath string
+)
+
+func FindWSL() string {
```

Comment:
Lets merge this but I am wondering if this should go into containers common logic?  We have similar things in there .. something like FindBinary...

Reply:
yeah it could probably reuse some more generic / common stuff but the search logic is a bit specialized in this case

Reply:
Using this here seems fine c/common is for user configuration and this seem rather hard coded? SO I don't see much value in giving users an option to overwrite the location?

Reply:
right yeah i agree that a user would never want/need to override this and it